monitoring/shutdown-javaagent/src/test/java/com/google/cloud/runtimes/monitoring/shutdown/AgentTest.java

```diff
@@ -0,0 +1,32 @@
+package com.google.cloud.runtimes.monitoring.shutdown;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.Map;
+
+public class AgentTest {
```

Comment:
FYI: test classes don't require javadocs, so I won't be commenting on the other test classes for that. :-)